source/chapters/setup.rst

```diff
@@ -4,36 +4,43 @@ Hardware Setup
 .. sectionauthor::
    T.Rafreider <trafreider@mixxx.org>
    S.Brandt <s.brandt@mixxx.org>
+   Be <be.0@gmx.com>
 
 This chapter describes the most common hardware setups to serve as examples. We
-will go over the setup of timecode records/CDs, MIDI control and keyboard
-control.
-
-Audio Output
-============
+will go over the setup of soundcards, controllers, and timecode records/CDs.
```

Comment:
Yes it is. You may have missed the "We" at the beginning of the sentence because you were looking at the diff.